distribution/docs/src/main/resources/content/_architectures/_queries/filterbuilder-api.adoc

```diff
@@ -7,7 +7,7 @@
 
 To avoid the complexities of working with the Filter interface directly and implementing the ${branding} Profile of the Filter specification, the ${ddf-catalog} includes an API, primarily in `${ddf-branding}.filter`, to build Filters using a fluent API.
 
-To use the `FilterBuilder` API, an instance of `${ddf-branding}.filter.FilterBuilder` should be used via the OSGi registry.
+To use the `FilterBuilder` API, an instance of `${ddf-branding}	.filter.FilterBuilder` should be used via the OSGi registry.
```

Comment:
❔ So we needed an intentional space here?

Reply:
nope. fixed